app/models/user.rb

```diff
@@ -343,6 +343,10 @@ def pro?
     end
   end
 
+  def vomitted?
+    Reaction.where(reactable_id: id, reactable_type: "User", category: "vomit", status: "confirmed").exists?
```

Comment:
You can type less with `Reaction.exists?(reactable_id: id, reactable_type: "User", category: "vomit", status: "confirmed")`. They issue the same query

Reply:
Thanks, nice suggestions! :+1: